packages/crypto/src/constants.ts

```diff
@@ -0,0 +1,30 @@
+export const SUITE_ID_1 = new Uint8Array([75, 69, 77, 0, 16]);
+export const SUITE_ID_2 = new Uint8Array([72, 80, 75, 69, 0, 16, 0, 1, 0, 2]);
+export const HPKE_VERSION = new Uint8Array([72, 80, 75, 69, 45, 118, 49]);
+// b"secret"
+export const LABEL_SECRET = new Uint8Array([115, 101, 99, 114, 101, 116]);
+// b"HPKE"
+// b"eae_prk"
+export const LABEL_EAE_PRK = new Uint8Array([101, 97, 101, 95, 112, 114, 107]);
+// b"shared_secret"
+// deno-fmt-ignore
+export const LABEL_SHARED_SECRET = new Uint8Array([
+  115, 104, 97, 114, 101, 100, 95, 115, 101, 99, 114, 101, 116,
+]);
+
+export const AES_KEY_INFO = new Uint8Array([
+  0, 32, 72, 80, 75, 69, 45, 118, 49, 72, 80, 75, 69, 0, 16, 0, 1, 0, 2, 107,
+  101, 121, 0, 143, 195, 174, 184, 50, 73, 10, 75, 90, 179, 228, 32, 35, 40,
+  125, 178, 154, 31, 75, 199, 194, 34, 192, 223, 34, 135, 39, 183, 10, 64, 33,
+  18, 47, 63, 4, 233, 32, 108, 209, 36, 19, 80, 53, 41, 180, 122, 198, 166, 48,
+  185, 46, 196, 207, 125, 35, 69, 8, 208, 175, 151, 113, 201, 158, 80,
+]);
+
+export const IV_INFO = new Uint8Array([
+  0, 12, 72, 80, 75, 69, 45, 118, 49, 72, 80, 75, 69, 0, 16, 0, 1, 0, 2, 98, 97,
+  115, 101, 95, 110, 111, 110, 99, 101, 0, 143, 195, 174, 184, 50, 73, 10, 75,
+  90, 179, 228, 32, 35, 40, 125, 178, 154, 31, 75, 199, 194, 34, 192, 223, 34,
+  135, 39, 183, 10, 64, 33, 18, 47, 63, 4, 233, 32, 108, 209, 36, 19, 80, 53,
+  41, 180, 122, 198, 166, 48, 185, 46, 196, 207, 125, 35, 69, 8, 208, 175, 151,
+  113, 201, 158, 80,
+]);
```

Comment:
it's be nice to have these constants in text / utf8 form and convert them to bytes as needed. For instance:
```js
const LABEL_SECRET = toUint8Array('secret')
const ...etc...
```

Also it'd be good to add comments for what these values are / why they can be hardcoded.